source/features/github-bugs.css

```diff
@@ -77,3 +77,21 @@
 .pagehead-actions :is(.btn, summary) .octicon {
 	margin-right: 4px !important;
 }
+
+/* Avoid excessive repo title wrap by deprioritizing the repo header actions #5398 */
+@media (min-width: 768px) { /* Match .d-md-inline-flex */
+	:root .pagehead-actions {
+		flex-shrink: 3 !important;
+		display: inline-flex !important;
+		flex-wrap: wrap;
+		justify-content: flex-end;
```

Comment:
```suggestion
		justify-content: flex-end;
		margin: 0 -8px -8px 0; /* Offset the unwanted padding caused by the margins on children elements */
```

Reply:
What for? I don't see any unwanted padding

Reply:
Ah I'll try it

Reply:
This directly overrides the margin we set above, CSS should ideally never undo itself.

I'll use `margin: 8px 8px 0 0;` to replace the existing value